.github/workflows/ci.yml

```diff
@@ -44,4 +44,6 @@ jobs:
       - uses: golangci/golangci-lint-action@v3.7.0
         with:
           version: v1.52.2
+          only-new-issues: true
+          skip-pkg-cache: true
```

Comment:
fixes an issue related to the linter spamming missing files if there is a linter problem on CI

Reply:
![image](https://github.com/onflow/flow-cli/assets/6364887/7f160a6b-f97d-4970-a8e7-c184a1eaddad)